.dockerignore

```diff
@@ -1,2 +1,4 @@
 docker-machine*
-.git
```

Comment:
Ignoring git would break `Version.Commit` initialization at compile time. Removing it so we get actual commit hash in machine versions.